### PR TITLE
Add Github actions for Pull Requests

### PR DESCRIPTION
**Pull request by breno12321**

### CODE REVIEW

File: .github/workflows/analysis.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: code-analysis
-on: [push]
+on: [push, pull_request]
```
Comment (Shivam010) [inline .github/workflows/analysis.yml:2]: Yeah, I forgot we kept it to be run only on push earlier. 

Now, I don't think there's any issue in adding pull requests into the scopes. Thanks

**Review by Shivam010 — APPROVED** [review]

**Review by Shivam010 — CHANGES_REQUESTED** [review]
@breno12321 
Let's continue the encoding issue in #57 and merge this here with only the workflow yaml changes.

Can you remove your changes from `rjs/helper.go` file, so that we can merge this?

**Comment by breno12321** [comment]
Oh sorry I thought I didn't sent them together

**Review by Shivam010 — APPROVED** [review]

**Comment by breno12321** [comment]
@Shivam010 Uhm why it still don't pass even though its just github actions change 🤔

**Comment by Shivam010** [comment]
It's the version issue, we are using the latest version of `redislabs/rejson` (i.e 2.0.6)

Redislabs have changed some command responses after `1.0.8` which is causing failures in our tests. I have run them locally with 1.0.8, they are working.

For now, limiting the support to 1.0.8 and merging changes

**Comment by Shivam010** [comment]
Ignoring the `code-analysis / staticcheck` too.

**Comment by breno12321** [comment]
Thanks <3

**Pull request merged**